requirements/test.in

```diff
@@ -7,3 +7,6 @@ pytest-django             # pytest extension for better Django support
 ddt                       # Run a test case multiple times with different input
 freezegun                 # Allows tests to mock the output of assorted datetime module functions
 mock
+
+more-itertools<6.0.0      # A dependency of pytest.  Pytest doesn't constrain it and 6.0.0 onward
+                          # only works with python 3
```

Comment:
This would also go in `constraints.txt`.

Reply:
Done.